README.rst

```diff
@@ -0,0 +1,28 @@
+=====
+Django data access log
+=====
+
+Django data access log is a simple Django app to log the access into a JRF viewset.
+
+
+Quick start
+-----------
+
+1. Add "data_access_log" to your INSTALLED_APPS setting like this::
+
+    INSTALLED_APPS = [
+        ...
+        'data_access_log',
+    ]
+
+2. Run `python manage.py migrate` to create the polls models.
+
+3. Add the `@logged` decorator to any viewset you'd like to log::
+
+    from data_access_log.decorators import logged
+
+    @logged
```

Comment:
what about @log_access ?

Reply:
Decorator should be a property of the function they modify (as far as i understand that's the usual way of doing it). Maybe `@logs_access`

Reply:
@log_data_access ?